contrib/completion/bash/docker

```diff
@@ -447,17 +462,20 @@ _docker_images() {
 			COMPREPLY=( $( compgen -W "true false" -- "${cur#=}" ) )
 			return
 			;;
+		*label=*)
+			return
+			;;
```

Comment:
I think these 3 lines are not need,  you didn't add this in `_docker_ps`,  why here?